Resources/Locale/en-US/_white/reagents/meta/fun.yml

```
@@ -0,0 +1,2 @@
+reagent-name-tranquility = tranquility
+reagent-desc-tranquility = Emits a strange aura of tranquility...
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)